packages/vscode-extension/README.md

```diff
@@ -1,64 +1,88 @@
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147328227.html" rel="noopener noreferrer" target=blank>Create SuiteScript Files</a>|Creates a SuiteScript file based on a script type template that also includes any modules you select to extend NetSuite and customize, search for, and process your data. 
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162938506015.html" rel="noopener noreferrer" target=blank>Create Project</a>|Creates SDF file-based projects to develop ACPs or SuiteApp projects for internal use or for commercial distribution.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147342366.html" rel="noopener noreferrer" target=blank>Deploy Project</a>|Deploys the folder containing the project. The project folder is zipped before deployment including only the files and folders referenced in the deploy.xml file. | Ctrl+Alt+D |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162930381001.html" rel="noopener noreferrer" target=blank>Import Files</a>|Imports files from a NetSuite account to your project. Note that you cannot import files from a SuiteApp.| Ctrl+Alt+F |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162971830372.html" rel="noopener noreferrer" target=blank>Import Objects</a>|Imports objects from a NetSuite account to your project.| Ctrl+Alt+O |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_.html" rel="noopener noreferrer" target=blank>List Files</a>|Lists the files existing in the File Cabinet of your account.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_163067396066.html" rel="noopener noreferrer" target=blank>List Objects</a>|Lists the SDF custom objects existing in your account.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147609118.html" rel="noopener noreferrer" target=blank>Set Up Account</a>|Adds the NetSuite accounts you want to customize. It also allows setting an account as default for a project.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=163067510539.html" rel="noopener noreferrer" target=blank>Update File from Account</a>.|Overwrites the selected file in the project with the matching file in an account.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147414469.html" rel="noopener noreferrer" target=blank>Upload File</a>|Uploads files to the FileCabinet folder without requiring the deployment of the entire project. Files must be in an SDF supported **FileCabinet** folder.| Ctrl+Alt+U |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147382361.html" rel="noopener noreferrer" target=blank>Update Object from Account</a>|Overwrites the custom objects in the project with the custom objects in an account.| Ctrl+Shift+U |
```

Comment:
I think we can just say?  (similar to file update)
Overwrites the selected object in the project with the matching object in an account.